kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -47,23 +94,35 @@ def create(
     env,
     target_path,
     jinja_file,
+    params,
 ):  # pylint: disable=too-many-locals,too-many-arguments
     """Create an Airflow DAG for a project"""
+    project_path = Path.cwd().resolve()
+    bootstrap_project(project_path)
+    with KedroSession.create(project_path=project_path, env=env) as session:
+        context = session.load_context()
+        dag_config = _load_config(context, pipeline_name)
+
+        # Update with params if provided
+        dag_config.update(params)
+
     jinja_file = Path(jinja_file).resolve()
     loader = jinja2.FileSystemLoader(jinja_file.parent)
     jinja_env = jinja2.Environment(autoescape=True, loader=loader, lstrip_blocks=True)
     jinja_env.filters["slugify"] = slugify
     template = jinja_env.get_template(jinja_file.name)
 
     package_name = metadata.package_name
-    dag_filename = f"{package_name}_dag.py"
+    dag_filename = f"{package_name}_{pipeline_name}_dag.py"
```

Comment:
Can we make it so that if no `--pipeline` name is passed, the file name remains how it was before. When I do `kedro airflow create` now, it creates `<project_name>___default___dag.py` which seems like a strange name.